### PR TITLE
Adjust ISM2400 frequencies according to v4 draft

### LINKED ISSUE #3067: ISM 2400 RX2 frequency should be 2.423 Ghz
#### Summary

Current ISM2400 RX2 frequency is `2422000000` ([here](https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.9/pkg/band/ism_2400.go#L129)) , as written in last regional parameter draft ([v4](https://members.lora-alliance.org/communities/community-home/digestviewer/viewthread?GroupId=127&MessageKey=a8df33ba-803a-4298-a30e-2e2ab8e0f7ec&CommunityKey=6114003d-ea3c-4b44-bf19-3b0e56817ce5&tab=digestviewer) - page 13) it should be `2423000000`

Thank you :)

### PR DESCRIPTION
**Pull request by rvolosatovs**
#### Summary

Closes #3067 
cc @Oliv4945 

#### Changes

- Set RX2 frequency equal to what is demanded by the spec
- Set beacon/ping slot frequency to what is demanded by the spec

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/band/ism_2400.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func init() {
 			MaxDataRate: ttnpb.DATA_RATE_7,
 		},
 	}
-	const beaconFrequency = 2422000000
+	const beaconFrequency = 2424000000
```
Comment (johanstokking) [inline pkg/band/ism_2400.go:45]: What is the source for this? It's not in https://github.com/TheThingsNetwork/lorawan-stack/issues/3067

cc @Oliv4945

Comment (rvolosatovs) [reply pkg/band/ism_2400.go:45]: This is from the v4 draft, please see the spec.

**Review by johanstokking — APPROVED** [review]

**Review by johanstokking — DISMISSED** [review]
Hold for breaking RX2 change

**Comment by rvolosatovs** [comment]
changed base to 3.10

**Review by htdvisser — APPROVED** [review]
Looks good to me 👍

**Pull request merged**